linera-execution/src/test_utils/solidity.rs

```diff
@@ -91,6 +98,16 @@ pub fn get_bytecode(source_code: &str, contract_name: &str) -> anyhow::Result<Ve
             writeln!(test_code_file, "{}", literal_path)?;
         }
     }
+    if source_code.contains("@openzeppelin") {
+        let _output = Command::new("npm")
```

Comment:
`let _output =` is not needed. Also, there must be a better method than `Command::output` if you don't need the output.